src/routes/+page.svelte

```diff
@@ -37,11 +34,10 @@
 		}}
 	/>
 
-    <!-- <Control position="top-right"/> -->
-    <NavigationControl position="top-right"/>
-    <GeolocateControl position="top-right"/>
-    <FullscreenControl position="top-right"/>
-
+	<!-- <Control position="top-right"/> -->
```

Comment:
Out of interest how to auto format/indent?

Reply:
in vs code, can use `ctrl + shift + i` to trigger the prettier formatter. can always run prettier from CLI too.

Reply:
Makes sense, and I use VS Code.